next-app/src/routes/projects/[project_code]/meta/+server.ts

```diff
@@ -1,80 +1,84 @@
-import { sf } from '$lib/fetch/server'
+import { sf } from '$lib/fetch/server';
 
 export async function get({ project_code, cookie }) {
-	const [
-		{ id, projectName: name, users },
-		{ entries, comments }
-	] = await Promise.all([
-		sf({ name: 'project_read_by_code', args: [ project_code ], cookie }),
-		sf({ name: 'lex_stats_all', args: [ project_code ], cookie }),
-	])
+  const [{ id, projectName: name, users }, { entries, comments }] =
```

Comment:
I'm not a fan of this change either...

Reply:
Here it is with `printWidth: 120`. Ultimately, I think with prettier we'd lose the power to add line-breaks wherever we want them.
![image](https://user-images.githubusercontent.com/12587509/195545883-8c49dcee-67d8-4b21-b8de-2f1dc34d7b66.png)

Reply:
yeah, that would go along way to lowering the friction for me of reading these things, thank you!